### ISSUE TITLE
Blurry area before animation finishes with QT windows with Kvantum

### ISSUE THREAD

**Issue by michaldybczak** [zzag/kwin-effects-yet-another-magic-lamp#70]
SUMMARY
***
When restoring window after minimization, with yet-another-magic-lamp I can see the blurry area where window will be right when animation starts. See the video.
This isn't a big problem, but it still looks a bit odd. See the video that explains it all.
***

STEPS TO REPRODUCE
1. Use Wayland session.
2. Set a transparent Kvantum theme like KvGlass, set Kvantum in application settings.
3. Use e yet-another-magic-lamp, The issue doesn't happen with system lamp effect.
4. Minimize and restore qt windows where there is a transparency (with blur), like Dolphin, Kalculator, Ksysguard and so on.

OBSERVED RESULT

Before the window fully restores, there is a blurry area before it. With other apps like Firefox, this issue doesn't happen.

EXPECTED RESULT

No blur area visible before the animation is finished.

SOFTWARE/OS VERSIONS
Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.3
KDE Frameworks Version: 5.104.0
Qt Version: 5.15.8
Kernel Version: 6.2.8-1-MANJARO (64-bit)
Graphics Platform: Wayland
Processors: 8 × Intel® Core™ i7-6700HQ CPU @ 2.60GHz
Memory: 7.6 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 530
Manufacturer: Alienware
Product Name: Alienware 17 R3
System Version: 1.13.1

[2023-03-27 12-52-25.webm](https://user-images.githubusercontent.com/10132958/227926855-5676b696-90d3-448d-a158-f9016ab42055.webm)

**Comment by farline99** [comment]
Yeah, kvantum does this things. Author can't do anything about it. I am sure.

**Comment by angrylinuxuser** [comment]
It's not only kvantum. I have the same thing with breeze (konsole with blur). Didn't update to much since 5.25 so i cant pinpoint is it the some change they had in the breeze theme while back or is it the effect

**Comment by orychalk** [comment]
Hello,
I tried many many themes (Kvantum, Aurorae, ...) but there is always the blur problem.
It's more visible on Kvantum (all the window area), and less on aurorae theme (transparent window area) but present too.
With the standard KDE Plasma Lamp effect, I don't see the problem.
I tried with all KDE Plasma 5.27.x series, but the only one solution, is to downgrade to 5.25.
Thanks

**Comment by GleammerRay** [comment]
This might be a problem that evolved in Kvantum, as it does not exist in older Kvantum versions because I have the same Kvantum theme installed on two different devices and only one of them has this problem.

I'm using Kvantum 0.18.0 and 1.0.7 and the older version does not seem to have this problem.

**Edit 1**: on second thought, it is indeed most probably a KDE Plasma 5.27 issue. Can anyone confirm if this is still the case with Plasma 6?

**Comment by GleammerRay** [comment]
Hello @michaldybczak , @farline99 , @angrylinuxuser , @orychalk  !

I have opened a pull request that resolves this issue over at #73.

#39 still occurs, but I find it much less annoying than this.
